mobile-widgets/qml/DivePlannerSetup.qml

```diff
@@ -10,7 +10,14 @@ import org.kde.kirigami 2.4 as Kirigami
 Kirigami.ScrollablePage {
 	title: qsTr("Dive planner setup")
 
-	property string textLength: (PrefUnits.length === "meters") ? qsTr("m/min") : qsTr("ft/min")
+	property string textLength: (PrefUnits.unit_system === "metric") ? qsTr("m/min") : qsTr("ft/min")
+//	property string textLength: PrefUnits.unit_system
+	onTextLengthChanged: {
+		console.log("--> New text << " + textLength )
+		spinAscrate75.value++
+		spinAscrate75.value--
+	}
```

Comment:
I don't understand what you are trying to do here. The increment/decrement looks like this is test code?

Reply:
it is not test code, added a larger comment as to why.